include/HubFramework/HUBViewController.h

```diff
@@ -128,6 +128,15 @@ NS_ASSUME_NONNULL_BEGIN
   didDisappearFromView:(UIView *)componentView;
 
 /**
+ *  Sent to a Hub Framework view controller's delegate when a component view will be reused
+ *
+ *  @param viewController The view controller in which a component disappeared
+ *  @param componentView The view that the component disappeared from
+ */
+- (void)viewController:(HUBViewController *)viewController
+         willReuseView:(UIView *)componentView;
```

Comment:
How about `willReuseComponentWithView:`, to make it clear that it's a component view, not the main view of the view controller?